src/staticFilesServer.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+
+function staticFilesServer() {
+  const server = http.createServer((request, response) => {
+    const path = request.url;
+
+    if (path === '/favicon.ico') {
+      response.statusCode = 204;
+      response.end();
```

Comment:
can be less 
```suggestion
     res.status(203).end();
```
check other places